src/mmw/js/src/analyze/views.js

```diff
@@ -57,6 +57,33 @@ var ResultsView = Marionette.LayoutView.extend({
             newProjectUrl = '/project/new/' + modelPackageName,
             projectUrl = '/project';
 
+        if (modelPackageName === 'gwlfe') {
```

Comment:
Good question.  I was a bit confused by the terminology used.  "No Land Cover" seemed to me to mean `NODATA` for that particular raster set, but it instead the code seems to mean, "more than just Open Water land cover type".  It appears that gwlfe requires more than 100% Open Water, if TR55 is the same, we should probably treat it the same.  It's possible that it just reports 100% runoff in that scenario.